beetmoverscript/utils.py

```diff
@@ -253,3 +253,7 @@ def get_creds(context):
 
 def get_bucket_name(context, product):
     return context.config['bucket_config'][context.bucket]['buckets'][product]
+
+
+def get_bucket_url_prefix(context):
+    return context.config['bucket_config'][context.bucket]['url_prefix']
```

Comment:
Hm, I suppose it's a feature that this will die if we try to push nightlies to the partner bucket. I'm wondering if we should add a schema for the `script_config.json` at some point.